jdbi/src/main/java/org/killbill/commons/jdbi/guice/DaoConfig.java

```diff
@@ -132,4 +132,9 @@
     @Config("org.killbill.dao.healthCheckExpected99thPercentile")
     @Default("50ms")
     TimeSpan getHealthCheckExpected99thPercentile();
+
+    @Description("Whether or not initialization should fail on error immediately")
+    @Config("org.killbill.dao.initializationFailFast")
+    @Default("false")
+    boolean isInitializationFailFast();
```

Comment:
Don't we want to default to true ?

Reply:
`true` is the current behavior -- it is less cloud friendly (timing issues).